en/docs/administer/product-administration/managing-users-and-roles/managing-user-roles.md

```diff
@@ -153,6 +131,75 @@ Follow the instructions below to create the `creator` , `publisher` and `subscri
 
     These roles do not have any permissions assigned to it, but it is used to manage the visibility of the corresponding service provider that is created in the format of `'<username>_<applicationName>_PRODUCTION'` within the Key Manager. The created service provider is only visible to users with the latter mentioned role that has been generated automatically. Only if a user with admin privileges assigns the latter mentioned role to a user, will that user be able to view the details of the service provider that is created per application.
 
+### Adding Role Mappings
+
+In the above example we mapped the new `creator` role to allow all operations allowed for `Internal/creator`. There we had to update each and every entry that contained `Internal/creator` role. This can be a tedious task when there are multiple scope mapping entries to be updated.
```

Comment:
```suggestion
In the above example, we mapped the new `creator` role to allow all operations allowed for `Internal/creator`. There we had to update each and every entry that contained the `Internal/creator` role. This can be a tedious task when there are multiple scope mapping entries to be updated.
```